fhservices.cfg.j2

```diff
@@ -15,6 +15,11 @@ define command {
        command_line   $USER1$/check_http -H $HOSTADDRESS$ -S
 }
 
+define command {
+  command_name   check_mongodb
+  command_line   /opt/rhmap/nagios/plugins/check_mongodb_wrapper.sh -A $ARG1$ -S $ARG2$ -w $ARG3$ -c $ARG4$
```

Comment:
The ARG1, ARG2, etc... are nagios conventions of how it handles the parameters inside it's config.  Those are populated by the parameters separated by !'s in the service's check_command.  I'm turning them into named parameters for the calll to the actual script, we can't make it simpler earlier than here, unfortunately.